neqo-transport/src/recovery/mod.rs

```diff
@@ -961,8 +964,8 @@ mod tests {
 
     const ON_SENT_SIZE: usize = 100;
     /// An initial RTT for using with `setup_lr`.
-    const TEST_RTT: Duration = ms(80);
-    const TEST_RTTVAR: Duration = ms(40);
+    const TEST_RTT: Duration = ms(7000);
+    const TEST_RTTVAR: Duration = ms(3500);
```

Comment:
Why these particular values? And why do they need to be that high?

Reply:
They came out of a discussion with @martinthomson at the Vancouver IETF